crates/vm/levm/src/vm.rs

```diff
@@ -1208,3 +1172,19 @@ fn get_number_of_topics(op: Opcode) -> Result<u8, VMError> {
 
     Ok(number_of_topics)
 }
+
+/// Gets account, first checking the cache and then the database (caching in the second case)
+pub fn get_account(cache: &mut CacheDB, db: &Arc<dyn Database>, address: Address) -> Account {
+    match cache::get_account(cache, &address) {
+        Some(acc) => acc.clone(),
+        None => {
+            let account_info = db.get_account_info(address);
+            let account = Account {
+                info: account_info,
+                storage: HashMap::new(),
```

Comment:
Hi Jere, quick question, why are we always creating a new storage every time we "get" an account?

Reply:
Because we actually want to get only the account info but manipulating the account itself is sometimes more comfortable. When we get the account info we want to insert that into cache, but it stores Accounts, not account info. So what we do is inserting the `Account` with the corresponding `AccountInfo` but with empty storage, and if we need the storage slot we have to access the Database.

Reply:
>  (...) and if we need the storage slot we have to access the Database.

Do you know where this is done? I am not quite clear on why we are doing it in two steps.